### PR TITLE
fix(services): fix preload.js not found problem when start to run

### PR DESCRIPTION
**Pull request by foolyoghurt**
### Env
node: v10.13.0
ndb: v1.0.42

image

### CODE REVIEW

File: services/ndd_service.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ class NddService {
       nodePath += process.platform === 'win32' ? ';' : ':';
     nodePath += path.join(__dirname, '..', 'lib', 'preload');
     const env = {
-      NODE_OPTIONS: `--require ndb/preload.js`,
+      NODE_OPTIONS: `--require ${path.join(nodePath, 'ndb/preload.js')}`,
```
Comment (ak239) [inline services/ndd_service.js]: I am not sure that this fix can resolve the problem. Do you have nice repro for this?
nodePath variable contains list of folders where node is looking for modules, this path plus ndb/preload.js most of the time would be something invalid.

Comment (adamjsimon) [reply services/ndd_service.js]: I am not sure if this fixes the other problem I run into with preload.js not found but if someone is already working on preload issues might be worth noting that I run into issues with preload.js not found on a new rails/webpacker setup... any basic out of the gate rails/webpacker install seems to exhibit the issue. Maybe this is the wrong place to mention it.

**Comment by adamjsimon** [comment]
I am not sure if this fixes the other problem I run into with preload.js not found but if someone is already working on preload issues might be worth noting that I run into issues with preload.js not found on a new rails/webpacker setup... any basic out of the gate rails/webpacker install seems to exhibit the issue. Maybe this is the wrong place to mention it.

**Comment by ak239** [comment]
Unfortunately, there is no activity in this pull request if you still can reproduce a problem - feel free to left comment here, open another pull request or file an issue.

**Pull request closed without merging**